src/Main.java

```diff
@@ -2,15 +2,25 @@
 import java.util.List;
 import java.util.Scanner;
 
+// Opdracht: Een main-klasse met een main-methode
+
+/* Vragen over de opdracht:
+
+1) Waar zijn de pokemons van de GymOwner gedefinieerd? Zijn dat nu dezelfde als die ik krijg als PokemonTrainer (hieronder gedefinieerd in main)? Zo ja, hoe kan ik de GymOwner andere pokemons toekennen?
+2) Ik heb nu voor mijn gevoel veel dezelfde code geschreven om health points af te trekken en te printen. Kan dit korter? (bijv. door methodes te gebruiken?) En zo ja, hoe?
```

Comment:
Nee voor nu is dit prima. 
ALs je dit korter zou willen moet je van te voren eerst goed de 'regels' duidelijk hebben welke variablen er allemaal zijn in het bepalen van hoeveel hp eraf gaan na 1 aanvalen. En als je dat duidelijk hebt zou je daar een functie van kunnen schrijven.